Doc/whatsnew/3.8.rst

```diff
@@ -1755,7 +1755,7 @@ The following features and APIs have been removed from Python 3.8:
 *  Starting with Python 3.3, importing ABCs from :mod:`collections` was
    deprecated, and importing should be done from :mod:`collections.abc`. Being
    able to import from collections was marked for removal in 3.8, but has been
-   delayed to 3.9. (See :issue:`36952`.)
+   delayed to 3.9. (See :issue:`36953`.)
```

Comment:
May as well avoid the redirect?
```suggestion
   delayed to 3.9. (See :gh:`81134`.)
```

Reply:
Oh nice! I didn't know that was a thing.